MoltenVK/MoltenVK/GPUObjects/MVKQueue.mm

```diff
@@ -524,7 +587,31 @@
 
 template <size_t N>
 void MVKQueueFullCommandBufferSubmission<N>::submitCommandBuffers() {
+	MVKDevice* mvkDev = getDevice();
+	uint64_t startTime = mvkDev->getPerformanceTimestamp();
+
 	for (auto& cb : _cmdBuffers) { cb->submit(this, &_encodingContext); }
+
+	mvkDev->addActivityPerformance(mvkDev->_performanceStatistics.queue.submitCommandBuffers, startTime);
+}
+
+template <size_t N>
+MVKQueueFullCommandBufferSubmission<N>::MVKQueueFullCommandBufferSubmission(MVKQueue* queue,
+																			const VkSubmitInfo* pSubmit,
+																			VkFence fence,
+																			MVKCommandUse cmdUse)
+	: MVKQueueCommandBufferSubmission(queue, pSubmit, fence, cmdUse) {
+
+	// pSubmit can be null if just tracking the fence alone
```

Comment:
Is this still true? I noticed above that you're now submitting an empty `MTLCommandBuffer` and waiting for that to finish.